arangod/GeneralServer/H2CommTask.cpp

```diff
@@ -53,6 +53,12 @@
 using namespace arangodb::basics;
 using std::string_view;
 
+namespace {
+constexpr std::string_view switchingProtocols(
+    "HTTP/1.1 101 Switching Protocols\r\nConnection: "
+    "Upgrade\r\nUpgrade: h2c\r\n\r\n");
```

Comment:
`h2c` identifies the protocol where HTTP/2 is run over cleartext TCP. What about HTTP2 over TLS?

Reply:
I won't make any functional additions in this PR, just fix the traffic accounting where possible.
I think HTTP/2 over TLS is negotiated using ALPN.